### PR TITLE
Preference screens

### PR DESCRIPTION
**Pull request by CodesByBaafogAyo**
Created Screen

### CODE REVIEW

**Review by Samsonroyal — APPROVED** [review]
Good to merge

**Comment by Samsonroyal** [comment]
> Good to merge

@bright2kwame awaiting your review to merge

**Review by bright2kwame — CHANGES_REQUESTED** [review]
button1, button2 can we give a more descriptive names here ??

File: clearpath/app/[dashboard]/preferences/page.css [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+  color: #64748B;
+  border: 1px solid #E2E8F0;
+  margin-bottom: 10px;
+  border-radius: 12px;
+  padding: 0 28px;
+  box-shadow: 0px 10px 10px -5px rgba(0, 0, 0, 0.04), 0px 20px 25px -5px rgba(0, 0, 0, 0.1);
+
+}
+
+.button2 {
```
Comment (bright2kwame) [inline clearpath/app/[dashboard]/preferences/page.css:140]: lets name this in a desriptive way

File: clearpath/app/[dashboard]/preferences/page.css [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+  color: #64748B;
+  border: 1px solid #E2E8F0;
+  margin-bottom: 10px;
+  border-radius: 12px;
+  padding: 20px 28px;
+  box-shadow: 0px 10px 10px -5px rgba(0, 0, 0, 0.04), 0px 20px 25px -5px rgba(0, 0, 0, 0.1);
+
+}
+
+.button {
```
Comment (bright2kwame) [inline clearpath/app/[dashboard]/preferences/page.css:158]: same here, lets name this descriptively

**Review by bright2kwame — CHANGES_REQUESTED** [review]

File: clearpath/app/[dashboard]/preferences/page.css (hunk 1 above)
Comment (CodesByBaafogAyo) [reply clearpath/app/[dashboard]/preferences/page.css:140]: Ok Bright

**Comment by Samsonroyal** [comment]
This PR has been resolved.

**Pull request closed without merging**